src/components/pages/Home/Pricing.tsx

```diff
@@ -23,16 +30,28 @@ const Pricing: FC = () => {
           Simple pricing for everyone
         </h2>
       </div>
-      <div className="grid max-w-2xl grid-cols-1 gap-y-10 sm:mx-auto lg:-mx-8 lg:max-w-3xl lg:grid-cols-2 lg:gap-x-8 lg:place-self-center xl:mx-0 xl:gap-x-8">
+      <div className="grid max-w-2xl grid-cols-1 gap-y-10 sm:mx-auto lg:-mx-8 lg:max-w-6xl lg:grid-cols-3 lg:gap-x-8 lg:place-self-center xl:mx-0 xl:gap-x-8">
+        <PricingCard
+          name="Basic"
+          price="$0/month"
+          cta="Sign up for free"
+          href={user?.uid ? "/tos" : "/sign-up"}
+          features={includedFeaturesForBasic}
```

Comment:
This plan allows showing last three searches, but in [this file](https://github.com/coreofscience/tos.coreofscience.org/blob/main/src/components/upload/TreeHistory/index.tsx#L7) we allow five, I think we will have to tweak it to ensure consistency.

Reply:
This tweak is not just about changing five for three, since the heigth of the div should be calculated based now on three elements, not five.